api/HTMLMarqueeElement.json

```diff
@@ -686,10 +686,10 @@
               "version_added": "4"
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
I'm guessing Opera 12.1 is from the collector. If so, can you update the description?

Reply:
I've tested http://mdn-bcd-collector.appspot.com/tests/api/HTMLMarqueeElement/trueSpeed in Opera 12.16 and 15 and it's only supported in 15. So this can't be from the collector, unless there's a bug.

Reply:
```suggestion
              "version_added": "15"
```